link/src/main/java/com/stripe/android/link/ui/LinkButtonView.kt

```diff
@@ -43,7 +43,7 @@ import com.stripe.android.link.theme.linkColors
 
 private val LinkButtonVerticalPadding = 6.dp
 private val LinkButtonHorizontalPadding = 10.dp
-private val LinkButtonShape = RoundedCornerShape(22.dp)
+private val LinkButtonShape = RoundedCornerShape(4.dp)
 private val LinkButtonEmailShape = RoundedCornerShape(16.dp) // Button corner radius - padding
```

Comment:
This need to be updated too now, I think.

Reply:
This is part of the Link sign up screen and has it's own UX, so I will leave it the same.

Reply:
No, it’s the shape of the email chip within the Link button. We need to update its corner radius to account for the new corner radius of its container.